website/src/content/docs/docs/emitters/sdk-clients/introduction.md

```diff
@@ -0,0 +1,134 @@
+  }
+```
+
+#### Note: Check for the Latest Version
+
+To ensure you are using the latest version of the packages, visit [npmjs](https://www.npmjs.com/) and search for the relevant packages.
+
+## Language-Specific Settings
+
+### JavaScript Client Emitter
+
+Generally no additional setting is required for JavaScript. However it's recommended to specify `packageDetails` which would provide package metadata in `package.json` and `README.md` files.
+
+#### packageDetails
+
+Provide the metadata for `package.json`, `README.md` information.
+
+| Property    | Description                                                            |
+| ----------- | ---------------------------------------------------------------------- |
+| name        | Package name used in `package.json`                                    |
+| description | Package description used in `package.json` file                        |
+| version     | Detailed version for your package, the default value is `1.0.0-beta.1` |
+
+Example configuration:
+
+```yaml
+packageDetails:
+  name: "${your_package_name}"
+  version: 1.0.0
+```
+
+### Python Client Emitter
+
+No additional setting needed for Python.
+
+### Java Client Emitter
+
+#### Required Dependencies
+
+Java (Java Development Kit) and Maven (Apache Maven) is required to be installed, before running Java client emitter.
+
+The required version and the URL for downloading is specified below:
+
+- [Java](https://docs.microsoft.com/java/openjdk/download) 17 or above. (Verify by running `java --version`)
+- [Maven](https://maven.apache.org/download.cgi). (Verify by running `mvn --version`)
+
+#### Configuration Options for Java
+
+No additional setting needed for Java.
+
+### .NET Client Emitter
+
+## Running Language-Specific Emitters in CLI
+
+1. Ensure that your package.json file is correctly configured to include the necessary dependencies for running the emitters
+
+1. Update the tspconfig.yaml file for properly configured for the language-specific emitter.
+
+```yaml
+emit:
+  - "@typespec/http-client-csharp"
+  - "@typespec/http-client-java"
+  - "@typespec/http-client-python"
+  - "@azure-tools/typespec-ts"
+options:
+  "@typespec/http-client-csharp":
+    emitter-output-dir: "{project-root}/../clients/dotnet"
+  "@typespec/http-client-java":
+    emitter-output-dir: "{project-root}/../clients/java"
+  "@typespec/http-client-python":
+    emitter-output-dir: "{project-root}/../clients/python"
+  "@azure-tools/typespec-ts":
+    emitter-output-dir: "{project-root}/../clients/javascript"
+    packageDetails:
+      name: "${your_package_name}"
+      version: 1.0.0
+```
+
+1. Once the package.json and tspconfig.yaml files are updated, you need to install all required dependencies.
+
+Run the following command:
+
+```bash
+tsp install
+```
+
+1. Run the emitter to compile your TypeScript code into the desired language. Use the following command to trigger the emitter and compile your project:
+
+```bash
+tsp compile .
```

Comment:
Should we clarify that this will only work if `main.tsp` is defined in the current directory? It might be better to specify the path to the tsp that you wish to compile.

Reply:
Updated.  For unbranded scenario,  main.tsp is the entry point for all emitters.